templates/_helpers.tpl

```diff
@@ -73,6 +73,18 @@ Create the name of the service account to use
 - name: GEOSERVER_DEBUG_INSTANCEID
   value: "true"
 {{- end }}
+{{- if .Values.geoserver.adminCredentials }}
```

Comment:
@jemacchi  this will be possible in values directly once we migrate to `helm-custom-pod`